packages/ui/src/solidity/AccountControls.svelte

```diff
@@ -129,9 +128,9 @@
       />
       Modules
       <HelpTooltip link="https://eips.ethereum.org/EIPS/eip-7579">
-        Enables a modular architecture where account functionality can be extended through external contracts (modules).
-        Supports validators for signature verification, executors for transaction handling, and fallback handlers for
-        additional features. Hooks are not supported by default.
+        Enables a modular architecture where account functionality can be extended through external contracts (modules)
+        and enhances batched execution. Supports validators for signature verification, executors for transaction
+        handling, and fallback handlers for additional features. Hooks are not supported by default.
```

Comment:
I would like to mention the installation and uninstallation flow, since this could potentially be the first encounter of devs with `ERC-7579 Modules`, and I think it helps building the mental model

```suggestion
        Enables a modular architecture where account functionality can be extended via installation and uninstallation of external contracts (modules)
        and enhances batched execution. Supports validators for signature verification, executors for transaction
        handling, and fallback handlers for additional features. Hooks are not supported by default.
```